source/features/hide-useless-comments.tsx

```diff
@@ -13,7 +13,7 @@ function unhide(event: delegate.Event<MouseEvent, HTMLButtonElement>): void {
 	}
 
 	// Expand all "similar comments" boxes
-	for (const similarCommentsExpandButton of select.all('.js-discussion .Details-content--closed > span:only-child')) {
+	for (const similarCommentsExpandButton of select.all('.pagination-loader-container .Details-content--closed > span:only-child')) {
```

Comment:
@yakov116 we can indeed simplify this selector to `'.js-discussion .Details-element:not([data-body-version]) > summary'`

Reply:
Thought so but it was waay past my bed time and I wanted to release it. (We don't want broken features)

I put it on my list for my next lint